src/elm/Community.elm

```diff
@@ -569,7 +570,8 @@ encodeUpdateLogoData c =
         , ( "invited_reward", Eos.encodeAsset c.invitedReward )
         , ( "has_objectives", Encode.int c.hasObjectives )
         , ( "has_shop", Encode.int c.hasShop )
-        , ( "has_kyc", Encode.int c.hasKyc )
+
+        -- , ( "has_kyc", Encode.int c.hasKyc )
```

Comment:
I guess we don't need this line. Also, maybe it's better to rename `encodeUpdateLogoData` funciton to `encodeUpdateCommunityData`?

Reply:
Actually at some point we will allow communities to enable KYC, so that's why I left it commented

Reply:
Oh, I see!